.tours/-welcome-to-e-bikes.tour

```diff
@@ -0,0 +1,72 @@
+{
+  "$schema": "https://aka.ms/codetour-schema",
+  "title": "👋 Welcome to E-Bikes",
+  "steps": [
+    {
+      "file": "force-app/main/default/lwc/productTileList/productTileList.js",
+      "description": "The Product Explorer lets you browse thought E-Bikes products by displaying cards that contain the product and it's price. It gets these by using an @wire to query call the getProducts Apex Class and providing the filter values that have been set. \n",
+      "line": 48
+    },
+    {
+      "file": "force-app/main/default/lwc/productCard/productCard.js",
+      "description": "The product card subscribes to the PRODUCT_SELECTED_MESSAGE message channel and passes the payload to the handleRecordLoaded method which will query for and return the selected record.",
```

Comment:
- Add code tag around `PRODUCT_SELECTED_MESSAGE`.
- Change "message channel" to "Lightning message channel" and add a link to https://developer.salesforce.com/docs/component-library/documentation/en/lwc/lwc.use_message_channel